pkg/api/http_server.go

```diff
@@ -234,6 +236,7 @@ func ProvideHTTPServer(opts ServerOptions, cfg *setting.Cfg, routeRegister routi
 		teamPermissionsService:       permissionsServices.GetTeamService(),
 		AlertNotificationService:     alertNotificationService,
 		DashboardsnapshotsService:    dashboardsnapshotsService,
+		PreferencesService:           preferencesService,
```

Comment:
It should be in lowercase. It's going to be internally used and it's not going to be exposed.